src/test/java/resources/channelCollection.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Channel Collection API test
+
+
+  Scenario Outline: Post channel collection API with existing BPMN flows
+    Given I have tenant as "<tenant>"
+    And I have the request body with payer ams identifier keys as "<key1>" and "<key2>"
+    When I call the channel collection API with client correlation id as 12345 expected status of 200
```

Comment:
please check the Idempotency feature,  If the same correlation ID is repeated, It will throw an error.

Reply:
Suggestion: use UUID